types/node/vm.d.ts

```diff
@@ -19,16 +22,26 @@ declare module "vm" {
          */
         columnOffset?: number;
     }
-    interface ScriptOptions extends BaseOptions {
-        displayErrors?: boolean;
-        timeout?: number;
-        cachedData?: Buffer;
+
+    interface ScriptOptionsObject extends BaseOptions {
+        cachedData?: Buffer | TypedArray | DataView;
+        /** @deprecated This option is deprecated in favor of script.createCachedData() */
         produceCachedData?: boolean;
+        /** This option is part of the experimental API for the --experimental-modules flag, and should not be considered stable. */
+        importModuleDynamically?: (specifier: string, module: SourceTextModule) => Object | SourceTextModule;
```

Comment:
Prefer to declare as method eg. `importModuleDynamically?(...): R`

Reply:
Done